internal/services/ingest.go

```diff
@@ -448,6 +492,10 @@ func (m *ingestService) collectLedgerTransactionData(ctx context.Context, getLed
 	if len(errs) > 0 {
 		return nil, fmt.Errorf("collecting ledger data: %w", errors.Join(errs...))
 	}
+	phaseDuration := time.Since(phaseStart).Seconds()
+	m.metricsService.ObserveIngestionPhaseDuration("collect_transaction_data", phaseDuration)
+	log.Ctx(ctx).Infof("🚧 Done collecting data from %d ledgers in %vs", len(ledgers), phaseDuration)
```

Comment:
I removed this log line and moved the phase duration to the main function. Any reason for moving it back?